pkg/provider/aws/edges/edges.go

```diff
@@ -0,0 +1,117 @@
+package edges
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/klothoplatform/klotho/pkg/core"
+	"github.com/klothoplatform/klotho/pkg/provider/aws/resources"
+)
+
+var KnowledgeBase = core.EdgeKB{
+	core.Edge{From: reflect.TypeOf(&resources.LambdaFunction{}), To: reflect.TypeOf(&resources.RdsInstance{})}: core.EdgeDetails{},
```

Comment:
Given the number of these we need to write, a helper like this might be useful:

```go
func typeOf[T core.Resource]() reflect.Type {
    var t T  
    return reflect.ValueOf(t).Type()
}

// e.g.
typeOf[*core.RdsInstance]()
```

Reply:
Good idea - could go a step further too:
```go
func NewEdge[Src Resource, Dest Resource]() Edge {
    return Edge{From: typeOf[Src](), To: typeOf[Dst]()}
}
```